tests/system/general/test_49_RCS_data_plotter.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+import shutil
+
+from ansys.aedt.core.visualization.advanced.rcs_visualization import MonostaticRCSData
+from ansys.aedt.core.visualization.advanced.rcs_visualization import MonostaticRCSPlotter
+from ansys.aedt.core.visualization.plot.matplotlib import ReportPlotter
+from ansys.tools.visualization_interface import Plotter
+import pandas as pd
+import pytest
+
+from tests import TESTS_GENERAL_PATH
+
+test_subfolder = "T49"
+
+
+@pytest.fixture(scope="module", autouse=True)
+def desktop():
+    """Override the desktop fixture to DO NOT open the Desktop when running this test class"""
+    return
+
+
+@pytest.fixture(scope="class")
+def setup_test_data(request, local_scratch):
+    """Fixture to set up the test data directory and file before running the test class."""
+    dir_original = os.path.join(TESTS_GENERAL_PATH, "example_models", test_subfolder)
+    data_dir = os.path.join(local_scratch.path, "rcs_files")
+    shutil.copytree(dir_original, data_dir)
+    request.cls.metadata_file = os.path.join(data_dir, "rcs_metadata.json")
```

Comment:
If I'm not wrong, this tests could be performed as unit test right ?
If that's the case, could we add them in such a way ? I can help you in doing so if you want :)

Reply:
@SMoraisAnsys I agree, it would be great if you could help me to define it, and then I can do the rest